pkgs/servers/misc/podgrab/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "podgrab";
+  version = "2021-04-14"; # No official version tags
+
+  src = fetchFromGitHub {
+    owner = "akhilrex";
+    repo = pname;
+    rev = "3179a875b8b638fb86d0e829d12a9761c1cd7f90";
+    sha256 = "sha256-vhxIm20ZUi+RusrAsSY54tv/D570/oMO5qLz9dNqgqo=";
+  };
+
+  vendorSha256 = "sha256-xY9xNuJhkWPgtqA/FBVIp7GuWOv+3nrz6l3vaZVLlIE=";
+
+  postInstall = ''
+    mkdir -p $out/share/
+    ln -s -r "$src/client" "$out/share/"
+    ln -s -r "$src/webassets" "$out/share/"
+  '';
+
```

Comment:
You probably want to add

```nix
passthru.tests = { inherit (nixosTests) podgrab; };
```

(and the corresponding nixosTests argument at the top)

Reply:
Sure. I saw this discussed on some other packages, what does it do exactly? A quick google search did not turn up any docs.

Reply:
The docs are here: https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes (look for `tests`, it's not actually a meta attribute)

passthru is effectively a set of attributes which don't affect the build of the package itself; passthru.tests means that the "tests" attribute set becomes available on the output package.

passthru.tests is a convention for finding relevant tests for a package.

Reply:
Alritght, thank you :-) !